positron/components/PositronCLH.js

```diff
@@ -97,7 +97,7 @@ PositronCLH.prototype = {
         };
         if (isLocal(resolvedURI)) {
           // If the URI is local, we are sure it won't wrongly inherit chrome privs
-          var features = "chrome,dialog=no,all";
+          var features = "chrome,dialog=no,all,resizable";
```

Comment:
Hmm, the Firefox Browser Toolbox equivalent is resizable, so I wonder why yours is not... I guess `all` implies resizable with Firefox. I'll try to see if I can find out why it differs here...